dist/OutSystemsUI.js

```diff
@@ -7304,6 +7305,21 @@ var OSFramework;
                     constructor(config) {
                         super(config);
                     }
+                    validateDefault(key, value) {
+                        let validatedValue = undefined;
+                        switch (key) {
+                            case OverflowMenu.Enum.Properties.Shape:
+                                validatedValue = this.validateInRange(value, OSUI.GlobalEnum.ShapeTypes.SoftRounded, OSUI.GlobalEnum.ShapeTypes.Sharp, OSUI.GlobalEnum.ShapeTypes.Rounded);
+                                break;
+                            case OverflowMenu.Enum.Properties.Position:
+                                validatedValue = this.validateString(value, OSUI.GlobalEnum.FloatingPosition.Auto);
```

Comment:
Why don't we use validateInRange here like we do for the shape property?

Reply:
Good point, I followed the same approach as in Notification. I will change it 👍

Reply:
Fixed!